Cargo.lock

```diff
@@ -6014,9 +6013,9 @@ dependencies = [
 
 [[package]]
 name = "sysinfo"
-version = "0.30.13"
+version = "0.30.11"
```

Comment:
Drunk `cargo` deciding to revert https://github.com/nushell/nushell/pull/13615

Reply:
Yes, it did. I continue to be amazed by "user-friendliness" of rust ecosystem :imp:

Reply:
Fixed.

Reply:
Thanks, that is so weird